c/tests/test_convert.c

```diff
@@ -73,9 +80,15 @@ test_single_tree_newick_errors(void)
     size_t buffer_size = 1024;
     char newick[buffer_size];
 
-
-    tsk_treeseq_from_text(&ts, 1, single_tree_ex_nodes, single_tree_ex_edges,
-            NULL, NULL, NULL, NULL, NULL);
+    tsk_treeseq_from_text(&ts,
+        1,
```

Comment:
This is a case where we don't want one-arg-per line I think.

Reply:
I'm new to this option.  Not sure if there are ways to say "break if args are short" or not.